org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/panel/InfoPanel.java

```diff
@@ -2450,7 +2450,8 @@ else if(IDesktop.ON_CLOSE_WINDOW_SHORTCUT_EVENT.equals(event.getName())) {
 	 */
 	protected void onCancel() {
 		// do not allow to close tab for Events.ON_CTRL_KEY event
-		SessionManager.getAppDesktop().setCloseTabWithShortcut(false);
+		if(isUseEscForTabClosing)
+			SessionManager.getAppDesktop().setCloseTabWithShortcut(false);
```

Comment:
shouldn't the same check applicable to other onCancel changes as well ?

Reply:
You're right, it should. I was able to reproduce the issue with opening a process modal dialog over an info window too.
I've added the logic to all types in the latest commit.